packages/components/src/Enumeration/Enumeration.component.js

```diff
@@ -48,26 +69,35 @@ function Enumeration({ displayMode, headerDefault, headerInput,
 	return (
 		<div className={enumerationClasses()}>
 			{getHeader()}
-			<Items items={items} itemsProp={itemsProp} currentEdit={currentEdit} />
+			<Items
+				items={items}
```

Comment:
This component should be also be consistent with the selection management of the List component (See DisplayTable.component)